### DISCUSSION TITLE
Next Major ToonSquid Update

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#480] [General]
Update 1.10.5 released the other day, and it's making me wonder when the next major update will come, or what update will it be, 1.11 or 2.0, but the biggest thoughts that are going through me is what new features will be in the next major update. Keiwan, if you're reading this, is there anyway you could give us a hint about what major features will come to ToonSquid in the next update? I feel like giving people that small bit of information would tide them over and get them excited at the same time.

**Comment by keiwando** [comment]
Generally speaking, this is the answer:
https://toonsquid.com/handbook/guides/faq/#next-update-when
